<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Geral</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>621</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Aut√¥mato Finito</string>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>511</width>
         <height>311</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>8</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>M1</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>M2</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>M3</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>M4</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>M5</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>M6</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>M7</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_10">
        <attribute name="title">
         <string>M8</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_11">
        <attribute name="title">
         <string>M9</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>*-&gt;S</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>a</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>S</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_new_column">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>60</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>+ coluna</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_delete_column">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>90</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>- coluna</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_new_row">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>120</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>+ linha</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_delete_row">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>150</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>- linha</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_final">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>180</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>final</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_not_final">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>210</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>n√£o final</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_determinize">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>350</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>determinizar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>350</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>complemento</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>380</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>M6 &lt;- M1‚à©M2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>350</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>minimizar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_copy_to_M1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>380</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>copiar para M1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_copy_to_M2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>380</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>copiar para M2</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Gram√°tica Regular</string>
      </property>
      <widget class="QTextEdit" name="textEdit_RG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>411</width>
         <height>201</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;A-&amp;gt;aA|&amp;amp;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Express√£o Regular</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_RE">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>411</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>bb*</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_RGtoFA">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>GR-&gt;AF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_REtoFA">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>ER-&gt;AF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pattern_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>GR ‚â° ER ?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Busca em texto</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1231</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Padr√£o em Texto</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_ER_pattern">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>631</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>( n√£o)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>125</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Express√£o Regular</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>39</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Texto</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_pattern_text">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>1081</width>
         <height>251</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A aranha arranha a r√£. A r√£ arranha a aranha. Nem a aranha arranha a r√£. Nem a r√£ arranha a aranha.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;O tempo perguntou pro tempo quanto tempo o tempo tem. O tempo respondeu pro tempo que o tempo tem tanto tempo quanto tempo o tempo tem.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;O peito do p√© de Pedro √© preto. Quem disser que o peito do p√© de Pedro √© preto, tem o peito do p√© mais preto do que o peito do p√© de Pedro.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;O peito do p√© de Pedro √© azul. Quem disser que o peito do p√© de Pedro √© azul, tem o peito do p√© mais azul do que o peito do p√© de Pedro.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ele disse: n√£o n√£o n√£o!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_pattern">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>buscar padr√£o</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>651</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Feedback de atividades</string>
    </property>
    <widget class="QTextEdit" name="textEdit_output">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>641</width>
       <height>151</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar </string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
